src/index.js

```diff
@@ -0,0 +1,195 @@
+const cityUpdate = () => {
+  const newCityText = state.cityTextField.value;
+  state.headerCity.textContent = newCityText;
+};
+
+const cityDefault = "Boston";
+
+const cityReset = () => {
+  state.cityTextField.value = cityDefault;
+  state.headerCity.textContent = state.cityTextField.value;
+};
+
+const landscapes = {
+  coldLandscape: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+  coolLandscape: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+  warmLandscape: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+  hotLandscape: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+};
+
+const skyOptions = {
+  sunny: "☁️ ☀️ ☁️ ☀️ ☀️  ☁️",
+  cloudy: "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+  rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+  snowy: "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+};
+
+const landscapeUpdate = () => {
```

Comment:
If we structure the landscape lookup as described above, we could refactor the `if`/`else if` chain in this function to loop through the data instead.